addons/mail/static/tests/discuss/call/call.test.js

```diff
@@ -859,3 +862,118 @@ test("should not show context menu on participant card when not in a call", asyn
+    const cameraStream1 = streams.at(-1);
+    expect(cameraStream1.getTracks()[0].readyState).toBe("live");
+    await click("[title='Stop camera']");
+    await contains(".o-discuss-CallParticipantCard video", { count: 0 });
+    await click("[title='Turn camera on']");
+    await contains(".o-discuss-CallParticipantCard video");
+    const cameraStream2 = streams.at(-1);
+    expect(cameraStream1.getTracks()[0].readyState).toBe("ended");
+    expect(cameraStream2.getTracks()[0].readyState).toBe("live");
+    await click("[title='Stop camera']");
+    await contains(".o-discuss-CallParticipantCard video", { count: 0 });
+    await click("[title='Share Screen']");
+    await contains(".o-discuss-CallParticipantCard video");
+    await contains(".o-mail-DiscussSidebarCallParticipants-status:contains('LIVE')");
+    const screenStream = streams.at(-1);
+    expect(screenStream.getTracks()[0].readyState).toBe("live");
+    expect(streams.length).toBe(4); // in total we requested 1 audio, 2 camera, 1 screen
+    // represents the unlink of a rtc session
+    pyEnv["bus.bus"]._sendone(serverState.partnerId, "mail.record/insert", {
+        "discuss.channel": [
+            {
+                id: channelId,
+                rtc_session_ids: [],
+            },
+        ],
+    });
+    await contains(".o-discuss-Call", { count: 0 });
+    expect(audioStream.getTracks()[0].readyState).toBe("ended");
+    expect(cameraStream1.getTracks()[0].readyState).toBe("ended");
+    expect(cameraStream2.getTracks()[0].readyState).toBe("ended");
+    expect(screenStream.getTracks()[0].readyState).toBe("ended");
+});
+
+test("network features", async () => {
+    const pyEnv = await startServer();
+    const channelId = pyEnv["discuss.channel"].create({ name: "General" });
+    const channelMemberId = pyEnv["discuss.channel.member"].create({
+        channel_id: channelId,
+        partner_id: pyEnv["res.partner"].create({ name: "Bob" }),
+    });
+    const env = await start();
+    const network = await makeMockRtcNetwork({ env, channelId });
+    const bobRemote = network.makeMockRemote(channelMemberId);
+    await openDiscuss(network.channelId);
+    await click("[title='Join Call']");
+    await contains(".o-discuss-CallParticipantCard[title='Bob']");
+    await bobRemote.updateConnectionState("connecting");
+    await contains(".o-discuss-CallParticipantCard[title='Bob'] .fa-exclamation-triangle");
+    await bobRemote.updateConnectionState("connected");
+    await contains("span[data-connection-state='connected']");
+    await bobRemote.updateUpload("screen", createVideoStream().getVideoTracks()[0]);
+    await contains(".o-discuss-CallParticipantCard[title='Bob'] video");
+    await bobRemote.updateInfo({ isRaisingHand: true });
+    await contains(".o-discuss-CallParticipantCard[title='Bob'] .fa-hand-paper-o");
+    await contains(".o-discuss-Call-notification:contains('Bob raised their hand')");
+});
```

Comment:
Test name is too generic, prefer small test that are business relevant for discuss calls.
Like:
- `Shows connecting state on cards`
- `Can see raised hands from other call participants`